in_toto/models/layout.py

```diff
@@ -177,12 +162,17 @@ def import_step_metadata_from_files_as_dict(self):
           pass
 
         else:
+
+          if 'signed' not in link_obj:
+            raise in_toto.exceptions.LinkNotFoundError("Invalid format. '{0}'"
+                " is not a valid in-toto metadata file.".format(filename))
```

Comment:
Can you add a unittest for this line?

Reply:
interesting you proposed adding `data.get("signed", {}).get("steps")` above but not here. Any reasoning behind this?

I'm reluctant to add a UT on this line as this feels like code that should be refactored (why don't we have a schema run on this?) why do we use schema anyway?

Reply:
Only negligible reasoning really. :) But yes, I thought about proposing the same solution here as well. What do you think?

The reason why we don't schema validate her is because we don't have link schema validation yet (which we should add https://github.com/in-toto/in-toto/issues/122).